contrib/platform/src/com/sun/jna/platform/win32/WlanApi.java

```diff
@@ -0,0 +1,188 @@
+package com.sun.jna.platform.win32;
+
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.PointerByReference;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Module Name:
+ *     wlanapi.h
+ * Abstract:
+ *     Definitions and data structures for wlan auto config client side API.
+ */
+public interface WlanApi extends Library {
+    WlanApi INSTANCE = Native.load("wlanapi", WlanApi.class);
+    int WLAN_MAX_NAME_LENGTH = 256;
+    int DOT11_SSID_MAX_LENGTH = 32; // 32 bytes
+
+    class WLAN_INTERFACE_INFO_LIST extends Structure {
+        public int dwNumberOfItems;
+        public int dwIndex;
+        public WLAN_INTERFACE_INFO[] InterfaceInfo = new WLAN_INTERFACE_INFO[1];
+
+        public WLAN_INTERFACE_INFO_LIST() {
+          setAutoSynch(false);
+        }
+
+        public WLAN_INTERFACE_INFO_LIST(Pointer p) {
+            super(p);
+        }
+
+        @Override
+        protected List<String> getFieldOrder() {
+            return Arrays.asList("dwNumberOfItems", "dwIndex", "InterfaceInfo");
+        }
+
```

Comment:
Please user `@FieldOrder` annotation. See https://github.com/eranl/jna/blob/4073685fec63fb1418ed42d5c0cf4037251f96aa/contrib/platform/src/com/sun/jna/platform/win32/WinCrypt.java#L51-L104 for an example.